content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
+description: An anaylsis of how cloud resources are utilized in ambient and sidecar service mesh architectures.
+publishdate: 2023-07-14
+attribution: "Greg Hanson - Solo.io"
+keywords: [istio,ambient,waypoint,ztunnel]
+---
+
+Istio Ambient Mesh is the new Istio data plane without requiring sidecars, refer to the [announcement](/blog/2022/introducing-ambient-mesh/) for more details. Istio Ambient Mesh is included in [Istio 1.18](https://github.com/istio/istio/releases/tag/1.18.0) but is still considered to be in [alpha feature status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Sidecars have been a staple of Istio’s architecture since day one and are responsible for the majority of features available in Istio today. However, sidecars require the injection of an additional container to each Kubernetes pod resource, each of which needs allocated resources from the pod.
+
+Many of you are familiar with the simplified operation brought by the Istio ambient architecture. Let’s explore how ambient mesh cuts the service mesh costs typically associated with sidecars.
+
+## Check Out the Savings
```

Comment:
less bolshy heading

You're about to mention ztunnel and waypoints; at this point I think it would pay to reintroduce them.  Don't assume people have read the Ambient blog.